assets/product_code.yml

```diff
@@ -25,3 +25,5 @@ product_code:
   efs-sh19: 40103
   efs-sh19-same-day: 40107
   sr01: 16033
+  confirmation-statement: 15000
+  ll-confirmation_statement: 15003
```

Comment:
once we do implement the ll-cs01, we will do it using hyphens for the filing type.
So this needs to be `ll-confirmation-statement`